apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/ResultStream.cs

```diff
@@ -131,16 +131,50 @@ private async Task<PartialResultSet> ComputeNextAsync(CancellationToken cancella
 
                 try
                 {
-                    if (_grpcCall == null)
+                    bool hasNext = false;
```

Comment:
It's a pity we have to have this here rather than within PooledSession. I assume it's just rather tricky to put it in PooledSession...